tests/compression_size_test.c

```diff
@@ -37,6 +37,8 @@
 #define	SAMPLE_RATE		16000
 #define	DATA_LENGTH		(SAMPLE_RATE)
 
+#define	CMP_TEST_PREFIX	"cmp"
```

Comment:
Let's move away from macros, which are ugly. Let's go with [Ulrich Drepper's suggestion](https://udrepper.livejournal.com/15119.html):
```suggestion
static const char CMP_TEST_PREFIX[] = "cmp";
```

Reply:
Ah that was a leftover from my original approach of prepending `*_TEST_PREFIX` to every filename, which quickly got out of hand. I'll change them over.